nuxeo-dm/nuxeo-platform-webapp-core/src/main/java/org/nuxeo/ecm/webapp/contentbrowser/WorkspaceActionsBean.java

```diff
@@ -267,7 +267,7 @@ public String createWorkspaceOld() {
 
             // duplicate the template
             String title = (String) tmpWorkspace.getProperty("dublincore", "title");
-            String name = IdUtils.generatePathSegment(title);
+            String name = Framework.getService(PathSegmentService.class).generatePathSegment(title);
```

Comment:
Remove the whole method `createWorkspaceOld` containing this as well, I don't think it's been used in like forever.